DEVELOPER.md

```diff
@@ -23,13 +23,14 @@ Current availability versions are:
 
 ```
 furyctl
-├── install : looks for a Furyfile (default `./Furyfile.yaml`) and download it's content`
+├── init : Downloads Furyfile.yml and kustomization.yaml from the distribution repository to the current directory.
+├── vendor : looks for a Furyfile (default `./Furyfile.yaml`) and download it's content`
 ├── printDefault : prints a Furyfile example
```

Comment:
I'd change this description to `prints an example Furyfile.yaml`
But it's also questionable whether we'll still need this, when we'll have `furyctl init`

Reply:
I've got the same doubt. Do we need it?

Reply:
We don't. I like to cancel stuff.

Reply:
I love to remove code!